burstphoto/cli.swift

```diff
@@ -15,11 +15,20 @@ struct MyProgram {
         
         // create a list of bursts to process
         let burst_dirs = [
-            //"/Users/martinmarek/Downloads/rafs mini/",
-            "/Users/martinmarek/My Drive/Coding/Burst/bursts/Fuji X-trans RAF/",
-            // "/Users/martinmarek/My Drive/Coding/Burst/bursts/33TJ_20150614_232110_642_dng/",
-            // "/Users/martinmarek/My Drive/Coding/Burst/bursts/Monika-HP-dng/",
-            // "/Users/martinmarek/My Drive/Coding/Burst/bursts/Monika-stars-DNG/",
+                     
+            "/Volumes/T7 2021/Workspace Photolab/Burst Photo Test/DNG_Child/Test/",
+            //"/Volumes/T7 2021/Workspace Photolab/Burst Photo Test/DNG_Child Kopie/",
+            //"/Volumes/T7 2021/Workspace Photolab/Burst Photo Test/DNG_Klavier/Test/",
+            //"/Volumes/T7 2021/Workspace Photolab/Burst Photo Test/burst002_iso1000/",
+            //"/Volumes/T7 2021/Workspace Photolab/Burst Photo Test/Moon/",
+            //"/Volumes/T7 2021/Workspace Photolab/Burst Photo Test/Dark Sky/",
+            //"/Volumes/T7 2021/Workspace Photolab/Burst Photo Test/dinosaur order/",
+            //"/Volumes/T7 2021/Workspace Photolab/Burst Photo Test/highlights/",
+            //"/Volumes/T7 2021/Workspace Photolab/Burst Photo Test/33TJ_20150614_232110_642_dng order/",
+            //"/Volumes/T7 2021/Workspace Photolab/Burst Photo Test/bench-1/",
+            //"/Volumes/T7 2021/Workspace Photolab/Burst Photo Test/martin 7/",
+            //"/Volumes/T7 2021/Workspace Photolab/Burst Photo Test/Jozef-1-dng/",
+            //"/Volumes/T7 2021/Workspace Photolab/Burst Photo Test/mountains-1/",
```

Comment:
I think this should be reverted since these are temporary development files. If you wish to keep them for local testing, I recommend creating a separate inactive changelist to keep track of these. Same comment for the xcuserdata

Reply:
We may even replace the paths with generic paths, such as "/Volume/My Burst Folder/ ... /"

Reply:
Resolved in commit [Generic CLI file](https://github.com/chris-rank/hdr-plus-swift/commit/7b974caefa4e8674ee1c8fb6993e40b03d5b00c7).